strings/tcg/set_two.json

```diff
@@ -103,12 +103,12 @@
 			"id": "praetorian",
 			"name": "Xenomorph Praetorian",
 			"desc": "The Praetorian is the Queen's royal guard, never seen far from the Queen's chambers.",
-			"rules": "{$Hivemind}. If you have 2 or more other Xeno cards on your field alongside this card, you may sacrifice 3 Xeno cards and add Xenomorph Queen to your hand from your deck.",
+			"rules": "{$Hivemind}. On Play: Add a Xenomorph Queen card to your hand from your deck.",
```

Comment:
on summon? what are the implications between play versus on summon?

Reply:
Was a mistake, changed to On Summon